### PR TITLE
Add newformat foucc defination

### PR DESCRIPTION
**Pull request by ljia5**
Add Y8/Y16/VYUY/YVYU/ARGB64/ABGR64 foucc defination

### CODE REVIEW

File: va/va.h [hunk 1]
```diff
@@ -3922,6 +3922,38 @@ VAStatus vaQuerySurfaceError(
  */
 #define VA_FOURCC_IYUV          0x56555949
 
+/** Y8: 8-bit greyscale.
+ */
+#define VA_FOURCC_Y8            0x20203859
```
Comment (FurongZhang) [inline va/va.h]: Is it possible for us to add the FourCC explanation as the above definition? Then, it is clear for the readers to know which FourCC. Thanks a lot in advance!

File: va/va.h [hunk 2]
```diff
@@ -3922,6 +3922,38 @@ VAStatus vaQuerySurfaceError(
...
+ *
+ * Four bytes per pair of pixels: Y, V, Y, U.
+ */
+#define VA_FOURCC_YVYU          0x55595659
+
+/** ARGB: packed 16-bit BGRA.
+ *
+ * Four bytes per pixel: alpha, red, green, blue.
+ */
+#define VA_FOURCC_ARGB64        0x34475241
```
Comment (FurongZhang) [inline va/va.h]: Is it possible for us to describe the bits for every channel? VA_FOURCC_ARGB16161616 which is following the above naming rules? Then, the name rules seems to be consistent. Thanks a lot in advance!

File: va/va.h [hunk 3]
```diff
@@ -3922,6 +3922,38 @@ VAStatus vaQuerySurfaceError(
...
+
+/** ABGR: packed 16-bit RGBA.
+ *
+ * Four bytes per pixel: alpha, blue, green, red.
+ */
+#define VA_FOURCC_ABGR64        0x34474241
```
Comment (FurongZhang) [inline va/va.h]: Same as the above.

File: va/va.h (hunk 2 above)
Comment (ljia5) [reply va/va.h]: change the command following the reviewed

File: va/va.h (hunk 3 above)
Comment (ljia5) [reply va/va.h]: change the command following the reviewed

**Comment by XinfengZhang** [comment]
@ljia5  I cant merge it , you have two commit , one commit is merge commit. please make sure your commit is clean, and rebase it

**Comment by ljia5** [comment]
commit clean and rebase.

**Pull request merged**